NOTICE.txt

```diff
@@ -4,3 +4,11 @@ Copyright 2023- The Apache Software Foundation
 This product includes software developed by The Apache Software
 Foundation (http://www.apache.org/).
 
+
```

Comment:
The format here is typically

product/project name
copyright
This project includes … 


More info on this at https://infra.apache.org/licensing-howto.html

Reply:
@michaelsembwever I think we don't need to add anything to NOTICE.txt after reviewing the link you sent, as well as what we do for Apache Cassandra. In fact, Cassandra has logback dependencies, but since they are not _bundled_, we don't need to add it to the notice.

From Cassandra's `build.xml`:
```
          <dependency groupId="ch.qos.logback" artifactId="logback-core" version="1.2.9"/>
          <dependency groupId="ch.qos.logback" artifactId="logback-classic" version="1.2.9"/>
```